src/main/java/io/strimzi/kafka/bridge/mqtt/core/MqttServer.java

```diff
@@ -21,7 +21,7 @@
  * Represents the MqttServer component.
  */
 public class MqttServer {
-    private static final Logger logger = LoggerFactory.getLogger(MqttServer.class);
+    private static final Logger log = LoggerFactory.getLogger(MqttServer.class);
```

Comment:
Normally, our checkstyles have rules that `static final` fields should be upper case.

Reply:
Interesting that even the HTTP bridge doesn't have this rule. I will check on the operator. Thanks!